chainermn/communicators/_base.py

```diff
@@ -36,8 +36,27 @@ def __init__(self, obj):
 
 class CommunicatorBase(object):
 
-    def __init__(self, mpi_comm):
+    def __init__(self, mpi_comm, use_nccl=False):
```

Comment:
If you change this default value to False, `PureNcclCommunicator.__init__()`, superclass initialization must be updated, too.
```python

        super(PureNcclCommunicator, self).__init__(mpi_comm)
        self._init_ranks()
```

Reply:
Fixed!

Reply:
And I added tests to check the values of intra_rank and inter_rank, in test_node_aware_communicator_base.py .